### PR TITLE
Filter places by Amenity

### PR DESCRIPTION
**Pull request by MrSakine**

### CODE REVIEW

File: web_dynamic/static/scripts/3-hbnb.js [hunk 1]
```diff
@@ -11,17 +11,17 @@ $(document).ready(function () {
     $('.amenities h4').text(nameAmenity.join(', '));
   });
 
-  $.get("http://localhost:5001/api/v1/status/", data => {
-    if (data.status == "OK") {
-      $('DIV#api_status').addClass("available");
+  $.get('http://0.0.0.0:5001/api/v1/status/', (res) => {
```
Comment (hamzahdili2001) [inline web_dynamic/static/scripts/3-hbnb.js:14]: you should leave localhost

Comment (MrSakine) [reply web_dynamic/static/scripts/3-hbnb.js:14]: Request `http://0.0.0.0:5001/api/v1/status/`

Comment (hamzahdili2001) [reply web_dynamic/static/scripts/3-hbnb.js:14]: you change it in my code too

Comment (MrSakine) [reply web_dynamic/static/scripts/3-hbnb.js:14]: I just make the scripts coherent since they are linked to each other

Comment (MrSakine) [reply web_dynamic/static/scripts/3-hbnb.js:14]: they are going to ask for the reviewer to check if we are using the right URL or not

**Review by hamzahdili2001 — APPROVED** [review]

**Pull request merged**